internal/fpath/path_windows_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package fpath
+
+import (
+	"testing"
+)
+
+func TestLocalPathWindows(t *testing.T) {
+	for i, tt := range []struct {
+		url  string
+		base string
+	}{
+		{
+			url:  `\`,
+			base: `\`,
```

Comment:
Shouldn't the base be here an empty string?

Reply:
https://golang.org/pkg/path/filepath/#Base

> If the path consists entirely of separators, Base returns a single separator.